src/sentry/constants.py

```diff
@@ -264,6 +264,15 @@ def get_all_languages() -> list[str]:
 # Default sort option for the group stream
 DEFAULT_SORT_OPTION = "date"
 
+# Default value for prevent AI config
+PREVENT_AI_CONFIG_GITHUB_DEFAULT = {
+    "org_defaults": {
+        "on_command_phrase": {"bug_prediction": True, "vanilla": True},
```

Comment:
What's `vanilla` in this context? Is there a more descriptive name for this sub-option we could provide here?